test/compute_bands.jl

```diff
@@ -125,9 +113,9 @@ if mpi_nprocs() == 1  # not easy to distribute
     for key in keys(ref_klabels)
         @test klabels[key] ≈ ref_klabels[key] atol=1e-15
     end
-
-    @test kpath[1] == [raw"\Gamma", "X", "W", "K", raw"\Gamma", "L", "U", "W", "L", "K"]
-    @test kpath[2] == ["U", "X"]
+    
+    @test kpath[1] = ["Γ", "X", "U"]
+    @test kpath[2] = ["K", "Γ", "L", "W", "X"]
```

Comment:
Why is this different?

Reply:
The [default path type choice for pymatgen seems to be the "SC" paths](https://github.com/materialsproject/pymatgen/blob/b789d74639aa851d7e5ee427a765d9fd5a8d1079/pymatgen/symmetry/bandstructure.py#L55) rather than SeeK-path/Hinuma (which is what Brillouin returns) - so that would cause a difference. The Hinuma paths are supposed to be more optimal, however.

Reply:
Ok thanks @thchr, that clarifies it. In that case I will make a commit with the Hinuma testdata from pymatgen to allow proper. comparison.